pkgs/development/tools/alloy/default.nix

```diff
@@ -44,7 +44,7 @@ let generic = { major, version, src }:
       homepage = "http://alloytools.org/";
       downloadPage = "http://alloytools.org/download.html";
       license = licenses.mit;
-      platforms = platforms.linux;
+      platforms = platforms.unix;
```

Comment:
Could you change the commit message to

```
alloy: enable build on Darwin
```

The format for commit messages is described in [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Reply:
Oops. Done.